paasta_tools/check_autoscaler_max_instances.py

```diff
@@ -124,28 +124,33 @@ async def check_max_instances(
-                        status = pysensu_yelp.Status.CRITICAL
-                        output = (
-                            f"{service}.{instance}: Service is at max_instances, and"
-                            " ratio of current value to target value"
-                            f" ({current_value} / {target_value}) is greater than the"
-                            " ratio of max_instances_alert_threshold to setpoint"
-                            f" ({threshold} / {setpoint})"
-                        )
-                    else:
-                        status = pysensu_yelp.Status.OK
-                        output = (
-                            f"{service}.{instance}: Service is at max_instances, but"
-                            " ratio of current value to target value"
-                            f" ({current_value} / {target_value}) is below the ratio of"
-                            f" max_instances_alert_threshold to setpoint ({threshold} /"
-                            f" {setpoint})"
-                        )
+                    try:
+                        current_value = suffixed_number_value(metric["current_value"])
+                        target_value = suffixed_number_value(metric["target_value"])
+
+                        if current_value / target_value > metric_threshold_target_ratio:
+                            status = pysensu_yelp.Status.CRITICAL
+                            output = (
+                                f"{service}.{instance}: Service is at max_instances, and"
+                                " ratio of current value to target value"
+                                f" ({current_value} / {target_value}) is greater than the"
+                                " ratio of max_instances_alert_threshold to setpoint"
+                                f" ({threshold} / {setpoint})"
+                            )
+                        else:
+                            status = pysensu_yelp.Status.OK
+                            output = (
+                                f"{service}.{instance}: Service is at max_instances, but"
+                                " ratio of current value to target value"
+                                f" ({current_value} / {target_value}) is below the ratio of"
+                                f" max_instances_alert_threshold to setpoint ({threshold} /"
+                                f" {setpoint})"
+                            )
+                    except KeyError:
+                        # we likely couldn't find values for the current metric from autoscaling status
+                        # if this is the only metric, we will return UNKNOWN+this error
+                        # suggest fixing their autoscaling config
+                        output = f'{service}.{instance}: Service is at max_instances, and there is an error fetching your {metrics_provider_config["type"]} metric.  Check your autoscaling configs or reach out to #paasta.'
```

Comment:
```suggestion
                        output = f'{service}.{instance}: Service is at max_instances, and there is an error fetching your {metrics_provider_config["type"]} metric. Check your autoscaling configs or reach out to #paasta.'
```

(you can ignore this, i just noticed that there's 2 spaces after the `.`)